packages/mds-compliance-service/repository/mappers.ts

```diff
@@ -33,7 +35,7 @@ export const ComplianceSnapshotEntityToDomain = ModelMapper<
   const { id, recorded, policy_id, policy_name, ...domain } = entity
   return {
     policy: {
-      policy_id,
+      policy_id: 'a-b-c-d', // TODO check if FIXME
```

Comment:
no default value here :)

Reply:
I remember now how this got there - I was talking with Neil about validating UUIDs and seeing what would trip the linter